lib/battle/ReachabilityInfo.h

```diff
@@ -27,7 +27,7 @@ struct DLL_LINKAGE ReachabilityInfo
 	{
 		const CStack * stack; //stack for which calculation is mage => not required (kept for debugging mostly), following variables are enough
 
-		bool attackerOwned;
+		ui8 side;

```

Comment:
@alexvins btw, any reason to change bool to ui8?  Enum is nice solution but not here, we should use struct called SideInBattle. Maybe you can make enum there instead of `std::array<SideInBattle, 2> sides;` This will for sure improve legibility of the code.